server/datastore/mysql/migrations/tables/20220627104817_AddHostBatteriesTable.go

```diff
@@ -0,0 +1,37 @@
+package tables
+
+import (
+	"database/sql"
+
+	"github.com/pkg/errors"
+)
+
+func init() {
+	MigrationClient.AddMigration(Up_20220627104817, Down_20220627104817)
+}
+
+func Up_20220627104817(tx *sql.Tx) error {
+	// there may be many batteries per host, so the primary key is an
+	// auto-increment, not the host_id.
+	_, err := tx.Exec(`
+CREATE TABLE host_batteries (
+  id            INT(10) UNSIGNED NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  host_id       INT(10) UNSIGNED NOT NULL,
+  serial_number VARCHAR(255) NOT NULL,
+  cycle_count   INT(10) NOT NULL,
+  health        VARCHAR(10) NOT NULL,
+  created_at    TIMESTAMP NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  updated_at    TIMESTAMP NULL DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+
+  UNIQUE KEY idx_host_batteries_host_id_serial_number (host_id, serial_number)
```

Comment:
A host can have multiple batteries?

Reply:
Didn't find that info in osquery's code, good find, that being said this is to future-proof the design, as there are devices with multiple batteries - we did have this discussion on the ticket here (and subsequent comments) : https://github.com/fleetdm/fleet/issues/4904#issuecomment-1167458243

Reply:
For example, Thinkpad T480 with dual batteries (discussed on lenovo's forum): https://forums.lenovo.com/t5/ThinkPad-T400-T500-and-newer-T-series-Laptops/Lenovo-Thinkpad-T480-both-batteries-not-charging/m-p/4587095

And Gnome support in settings for multiple batteries (discussed in a Pop_OS github issue): https://github.com/pop-os/gnome-control-center/issues/101#issuecomment-697644577

Reply:
Interesting!

(Also "both batteries not charging" just sounds funny and ironic)